geoopt/manifolds/poincare/math.py

```diff
@@ -509,7 +513,7 @@ def _dist0(x, c, keepdim: bool = False, dim: int = -1):
     return dist_c * 2 / sqrt_c
 
 
-def clip_tangent(x, u, *, c=1.0, dim=-1):
+def clip_tangent(x, u, *, c=1.0, dim=-1, eps=None):
```

Comment:
`eps` could be smth interpretable, e.g. "result is such that adding a parallel vector of length `L` increases absolute velocity at least by `f(L, eps)`

Reply:
I removed this useless function